src/main.rs

```diff
@@ -260,10 +261,10 @@ fn run(matches: &clap::ArgMatches) -> Result<()> {
             proj.run(&runner, service, cmd.as_ref(), &opts)?;
         }
         "run-script" => {
-            warn_if_pods_are_enabled_but_not_running(&proj)?;
```

Comment:
This is a tricky issue. This warning is a major frustration-saver, and it's intended for the multiple-pod case. My recommendation: Put this warning back, and disable it only when `--no-deps` is passed.

Reply:
I didn't think this warning made sense - we don't expect any pods to be running before we execute `run` on them.

Reply:
@dkastner Hmm, it's in `run-script`, so maybe you're right. But it has been a _huge_ issue for `run` in the past, because when you run `run diamond` or something, the database needs to be up.

The `run` version of this warning was based on user testing and experience, and we should definitely keep it. For `run-scripts`, I would argue:

- If you pass `--no-deps`, then we shouldn't warn.
- If you _don't_ pass `--no-deps`, then you want the dependencies to be running. Docker will handle dependencies _inside_ the same pod for you, but it doesn't know about cross pod stuff. This warning exists to make sure that your cross-pod dependencies are in a reasonable state.

So after thinking about it, if somebody doesn't pass `--no-deps`, we should almost certainly warn about missing cross-pod dependencies for consistency with Docker's underlying choice to _start_ the containers it knows about.

Reply:
ok i'll just leave it in then, because in reality the no-deps feature only applies to containers within the same pod. There may likely be dependencies on other pods, like the db.

Reply:
OK, cool, then we're clear to merge. Thank you!